modules/network-observability-quickfilter.adoc

```diff
@@ -11,19 +11,22 @@ Query Options::
 You can use *Query Options* to optimize the search results, as listed below:
 
 ** *Log Type*: The available options *Conversation* and *Flows* provide the ability to query flows by log type, such as flow log, new conversation, completed conversation, and a heartbeat, which is a periodic record with updates for long conversations. A conversation is an aggregation of flows between the same peers.  
-** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for the *Overview* and *Topology* view. The default selected value is *Destination*.
+** *Duplicated flows*: A flow might be reported from several interfaces, and from both source and destination nodes, making it appear in the data several times. By selecting this query option, you can choose to show duplicates. Duplicates are hidden by default. Use the *Direction* filter in the *Common* section in the to switch between ingress and egress traffic.
```

Comment:
A word seems missing in the last sentence: `Use the Direction filter in the Common section in the (filters dropdown?) to switch between ingress and egress traffic`

Reply:
Yep you're right I didn't finish this phrase. I think I was experimenting with how to phrase filter dropdowns and went below to advanced filters and forgot to come back.